source/type/pb_converter.h

```diff
@@ -180,20 +180,235 @@ inline pbnavitia::CommercialMode* get_sub_object(const nt::CommercialMode*,
 inline pbnavitia::Trip* get_sub_object(const nt::MetaVehicleJourney*,
                                        pbnavitia::PtObject* pt_object) { return pt_object->mutable_trip(); }
 
+namespace {
+
+template<typename Pb> struct type_trait;
+template<> struct type_trait<nt::Line> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Line>*  type;
+};
+template<> struct type_trait<nt::ValidityPattern> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::ValidityPattern>*  type;
+};
+template<> struct type_trait<nt::StopArea> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::StopArea>*  type;
+};
+template<> struct type_trait<nt::StopPoint> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::StopPoint>*  type;
+};
+template<> struct type_trait<nt::Network> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Network>*  type;
+};
+template<> struct type_trait<nt::Route> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Route>*  type;
+};
+template<> struct type_trait<nt::Company> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Company>*  type;
+};
+template<> struct type_trait<nt::CommercialMode> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::CommercialMode>*  type;
+};
+template<> struct type_trait<nt::PhysicalMode> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::PhysicalMode>*  type;
+};
+template<> struct type_trait<nt::LineGroup> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::LineGroup>*  type;
+};
+template<> struct type_trait<ng::POI> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Poi>*  type;
+};
+template<> struct type_trait<ng::POIType> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::PoiType>*  type;
+};
+template<> struct type_trait<nt::VehicleJourney> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::VehicleJourney>*  type;
+};
+template<> struct type_trait<nt::Calendar> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Calendar>*  type;
+};
+template<> struct type_trait<nt::Contributor> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Contributor>*  type;
+};
+template<> struct type_trait<nt::Frame> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Frame>*  type;
+};
+template<> struct type_trait<nt::StopPointConnection> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Connection>*  type;
+};
+template<> struct type_trait<nt::MetaVehicleJourney> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Trip>*  type;
+};
+
+template<typename Pb>
```

Comment:
All the `get_mutable`, `get_pb_type`, `get_type`, and even `type_trait` can be moved to the cpp.
(In fact all of this anonymous namespace)
This avoids multiple clang warnings on unused methods.